web/content/index.adoc

```diff
@@ -27,6 +27,8 @@ The following releases are supported, meaning they receive (security) updates.
 
 The future version is built in nightly.
 
+* link:/release/3.10/[Foreman 3.10 & Katello 4.12]
```

Comment:
Please move this below line 32 and drop the `// ` so that there's a short introduction to the 3.10 RC.